build.sbt

```diff
@@ -1,9 +1,25 @@
 import sbt.Keys._
 
+import  org.scalajs.sbtplugin.ScalaJSCrossVersion
+
 import org.scalajs.jsenv.selenium.SeleniumJSEnv
 import org.scalajs.jsenv.selenium.Firefox
 import org.scalajs.jsenv.selenium.CustomFileMaterializer
 
+import com.typesafe.tools.mima.plugin.MimaPlugin.mimaDefaultSettings
+import com.typesafe.tools.mima.plugin.MimaKeys.{previousArtifact, binaryIssueFilters}
+
+val previousVersion = "0.1.1"
+val previousSJSBinaryVersion =
+  ScalaJSCrossVersion.binaryScalaJSVersion(previousVersion)
```

Comment:
This does not make any sense. `binaryScalaJSVersion` expects a version number *of Scala.js*, not of an arbitrary library. Fortunately, you don't need it ;)